File: Data Structures & Algorithms/kth-smallest-integer-in-bst/submission-0.cs
/**
 * Definition for a binary tree node.
 * public class TreeNode {
 *     public int val;
 *     public TreeNode left;
 *     public TreeNode right;
 *     public TreeNode(int val=0, TreeNode left=null, TreeNode right=null) {
 *         this.val = val;
 *         this.left = left;
 *         this.right = right;
 *     }
 * }
 */

public class Solution {
    public int KthSmallest(TreeNode root, int k) 
    {
        int count = 0;
        int returnval = 0;
        DFS(root);
        return returnval;
        void DFS(TreeNode node)
        {
            if(node == null)
            {
                return;
            }
            DFS(node.left);
            count++;
            if(count == k)
            {
                returnval = node.val;
            }
            DFS(node.right);
        }
    }
}

File: Data Structures & Algorithms/append-characters-to-string-to-make-subsequence/submission-0.java
class Solution {
    public int appendCharacters(String s, String t) 
    {
        int i = 0;
        int j = 0; 
        while(i < s.length())
        {
            if(j < t.length() && s.charAt(i) == t.charAt(j))
            {
                j++;
            }
            i++;
        }
        int returnval = 0;
        while(j < t.length())
        {
            j++;
            returnval++;
        }
        return returnval;
    }
}